src/bitset.cpp

```diff
@@ -75,6 +75,14 @@ void bitset_or(
     (*a) |= (*b);
 }
 
+//[[Rcpp::export]]
+void bitset_xor(
+    const Rcpp::XPtr<individual_index_t> a,
+    const Rcpp::XPtr<individual_index_t> b
+    ) {
+    (*a) &= ~(*b);
```

Comment:
This appears to implement

```r
a$and(b$not())
```

Which _is_ set difference, but is not _xor_.

Which do you want to implement? Both isn't a bad idea.

To implement xor I would do 
```suggestion
    (*a) ^= *b;
```

To do set difference, I'd add something like this to the Bitset R6 object:

```r
difference = function(other) self$and(other$not())
```

Reply:
Oh interesting! Yeah I think both is a good idea. I'll work that one out.

Reply:
"xor" properly renamed "set_difference", and proper "xor" added. Included more significant tests for both, for disjoint, identical, and partially overlapping sets.